experiment/pretest.json

```diff
@@ -0,0 +1,92 @@
+[
+{"question":"1. Mass spectrum gives valuable information about:",
+"answers":{
+            "a":"molecular mass",
+            "b":"functionals groups",
+            "c":"conjugation",
+            "d":"vibrational states "
+          },
+"correctAnswer":"a"},
+
+{"question":"2. The ions are separated based on their mass to charge ratios in",
+"answers":{
+            "a":"ionsiation chamber",
+            "b":"mass analyzer",
+            "c":"sample inlet",
+            "d":"detector"
+          },
+"correctAnswer":"c"},
+
+{"question":"3. The typical ionisation energy used in a EI-MS is:",
+"answers":{
+            "a":"0.7 eV",
+            "b":"7 eV",
+            "c":"70 eV",
+            "d":"700 e V"
+          },
+"correctAnswer":"a"},
+
+{"question":"4. Mass spectrum is a plot of:",
+"answers":{
+            "a":"relative intensity vs mass-to-charge",
+            "b":"relative intensity vs mass-to-energy",
+            "c":"voltage vs mass-to-charge",
+            "d":"voltage vs mass -to -energy"
+          },
+"correctAnswer":"c"},
+
+{"question":"5. The molecular ion peak in a mass spectrum corresponds to:",
+"answers":{
+            "a":"ion formed by fragmentation of a molecule",
+            "b":"molecule losing an electron to form a radical cation",
+            "c":"the most abundant ion",
+            "d":"ion with smallest mass-to-charge"
+          },
+"correctAnswer":"a"},
+
+{"question":"6. The base peak in a mass spectrum corresponds to:",
+"answers":{
+            "a":"ion formed by fragmentation of a molecule",
+            "b":"molecule losing an electron to form a radical cation",
+            "c":"the most abundant ion",
+            "d":"ion with smallest mass-to-charge"
+          },
+"correctAnswer":"c"},
+
+{"question":"7. In an magnetic sector mass analyser, the ion separation is based on:",
+"answers":{
+            "a":"ions with the correct mass-to-charge ratio undergo stable oscillation and passes through the cylinderal rods before they hit the detector",
```

Comment:
'pass through' not 'passes through'

Reply:
Sir we can ignore the pretest file for the time being because all the questions will be updated when the SME sends her question set.